src/EditorFeatures/Core/GoToDefinition/AbstractGoToSymbolService.cs

```diff
@@ -6,16 +6,18 @@
 
 namespace Microsoft.CodeAnalysis.Editor.GoToDefinition
 {
+    // Ctrl+Click (GoToSymbol)
     internal abstract class AbstractGoToSymbolService : ForegroundThreadAffinitizedObject, IGoToSymbolService
     {
         public async Task GetSymbolsAsync(GoToSymbolContext context)
         {
             var document = context.Document;
             var position = context.Position;
             var cancellationToken = context.CancellationToken;
-
             var service = document.GetLanguageService<IGoToDefinitionSymbolService>();
-            var (symbol, span) = await service.GetSymbolAndBoundSpanAsync(document, position, cancellationToken).ConfigureAwait(false);
+
+            // includeType: if the token has no aliased, referenced or declared symbols (mostly literals) we want to disable Ctrl+Click on them
```

Comment:
> if the token has no aliased, referenced or declared symbols

a lot of tokens have none of those, maybe it would be better to add "but has a type symbol"?

Reply:
what's your suggestion for the full message?

Reply:
i don't know. maybe it's fine. i just found it a little confusing to say "if the token has no aliased, referenced or declared symbols (mostly literals)",
because that includes the majority of tokens that have no symbol whatsoever, not even a type.

Reply:
// includeType: enable Ctrl+Click only if the token has aliased, referenced or declared symbols

Reply:
what I would have probably written is this:
> we only want ctrl+click on tokens that have an actual aliased, referenced or declared symbol
if the token has none of those but does have a type (mostly literals), we're not interested

but don't take my word as gospel

Reply:
this one is actually quite good, I'll use it